examples/test_sqg.py

```diff
@@ -5,9 +5,9 @@
 
 # the model object
 year = 1.
-m = sqg_model.SQGModel(L=2.*pi,nx=128, tmax = 100*year,
-        beta = 0., Nb = 1., H = 1., rek = 0., rd = None, dt = 0.01,
-                     taveint=year, use_fftw=True, ntd=8)
+m = sqg_model.SQGModel(L=2.*pi,nx=512, tmax = 40.005,
+        beta = 0., Nb = 1., H = 1., rek = 0., rd = None, dt = 0.005,
+                     taveint=1, ntd=8)
```

Comment:
Ok, 8 is probably over the top.  My laptop only has 4 so I don't know where I got this from.  That being said, your barotropic example uses 4 and so using 4 above seems consistent.  
Now that I'm using 512x512, when I use 4 threads, it uses almost 3 of them, which is pretty good I think and does speed things up.